internal/modules/mint/test.py

```diff
@@ -0,0 +1,64 @@
+"""
+This module contains all mint test functions.
+"""
+import logging
+import time
+import json
+import unittest
+from modules.mint.query import(
+    query_annual_provision, 
+    query_inflation, 
+    query_params
+)
+from internal.utils import env
+
+HOME = env.HOME
+DAEMON_HOME = env.DAEMON_HOME
+logging.basicConfig(format="%(message)s", level=logging.DEBUG)
+
+logging.info("INFO :: Running mint module test scripts")
+
+
+class TestMintModuleQueries(unittest.TestCase):
+    """
+    The class `TestMintModuleQueries` contains test functions
+    which test queries of mint module
+    """
+
+    def test_params_query(self):
+        """
+        The function `test_params_account` will query
+        mint module params and test with genesis params
+        """
+        # query params
+        path = f"{HOME}/.simd-1/config/"
+        with open(path + "genesis.json",encoding="utf8") as file:
+            data = json.load(file)
+
+        query_param = query_params()[1]
+
+        assert data["app_state"]["mint"]["params"] == query_param, "missmatch in params"
+        time.sleep(3)
```

Comment:
We don't need to match with genesis value. There could be other tests which updates genesis params once the chain started. Just query the params, and make sure the result is expected (not error, should be an integer result etc). Also let's avoid function descriptions for tests. not required